### DISCUSSION TITLE
TUYA Zigbee: How can I Read a zigbee value

### DISCUSSION THREAD

**Discussion by pickonedev** [arendst/Tasmota#11701] [Q&A] [answered]
Hi guys!

If I can request to set the temperature with this command `zbsend {"Device":"Computer_Heater","Write":{"TuyaTempTarget":70}}`, what command I should use to request the value which is already set? I tried wit `zbsend {"Device":"Computer_Heater","Read":{"TuyaTempTarget"}}` but it is not working.

Thank you in advance!

**Comment by barbudor** [comment]
I'm not using Zigbee myself but the JSON syntax you used is wrong as the key "TuyaTempTarget" as no value.
At least I would try `zbsend {"Device":"Computer_Heater","Read":"TuyaTempTarget"}`

**Reply by pickonedev** [reply]
I tried this as well, but I get `stat/ZigBee_Bridge/RESULT = {"ZbSend":"Missing parameters"}`

**Comment by sfromis** [comment, marked as answer]
Not at all sure about this, but try:
`zbsend {"Device":"Computer_Heater","Read":{"TuyaTempTarget":true}}`

**Reply by pickonedev** [reply]
No luck :-(

**Reply by sfromis** [reply]
For the record, the syntax I suggested is generally the correct one, this worked for a Zigbee bulb:
`{"device":"Ikea-Opal-1","read":{"CT":true}}`
This of course depends on the Zigbee device cooperating, and unfortunately Tuya devices misuse Zigbee to not follow standards. (Lower vs mixed case for keys like `Device` and `Read` does not matter)

**Reply by pickonedev** [reply]
Yes, I think that's the problem :-( Using tuya...

I tried this zbsend read command on other zigbee devices (no ikea), and still not working
